### PR TITLE
add disco_diffusion_cnclip_vitb16 module

### PR DESCRIPTION
**Pull request by rainyfly**
This module is developed based on [discoart](https://github.com/jina-ai/discoart), which uses guided denoising diffussion model to generate images semantically close to input text prompts. Here, Chinese-clip model is used to guide generation process.

### CODE REVIEW

File: modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+## 一、模型基本信息
+
+### 应用效果展示
+
+   - 输入文本 "小桥流水人家"
+
+   - 输出图像
+   <p align="center">
+     <img src="https://user-images.githubusercontent.com/22424850/183022413-7906d427-a7dc-40a1-af2c-53e571b89a09.png"  width = "80%" hspace='10'/>
```
Comment (nepeplwu) [inline modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md]: 建议替换一个更好的效果图

File: modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      - steps(Optional[int]): 生成图像的完整迭代次数。
+      - init_scale(Optional[int]): 控制生成和初始图像相似的相似度，该值在指定初始图像时候起作用，值越大，越贴近初始图像。
+      - perlin_init(Optional[bool]): 是否使用perlin噪声作为初始输入，默认为False。使用perlin噪声作为初始图像对于纹理有比较好的生成效果。
+      - perlin_mode(Optional[str]):  perlin噪声的模式，可选值为"colored", "gray"或者"mix"。
+      - seed(Optional[int]): 随机种子，由于输入默认是随机高斯噪声，设置不同的随机种子会由不同的初始输入，从而最终生成不同的结果，可以设置该参数来获得不同的输出图像。
+      - output_dir(Optional[str]): 保存输出图像的目录，默认为"disco_diffusion_cnclip_vitb16_out"。
+
+
+    - **返回**
+      - ra(DocumentArray): DocumentArray对象， 包含`n_batches`个Documents，其中每个Document都保存了迭代过程的所有中间结果。详细可参考[DocumentArray使用文档](https://docarray.jina.ai/fundamentals/documentarray/index.html)。
```
Comment (nepeplwu) [inline modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md:104]: ra是什么

Comment (rainyfly) [reply modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md:104]: result array

File: modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md (hunk 1 above)
Comment (rainyfly) [reply modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md]: Done

File: modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md (hunk 2 above)
Comment (rainyfly) [reply modules/image/text_to_image/disco_diffusion_cnclip_vitb16/README.md:104]: Done

**Pull request merged**